README.md

```diff
@@ -1,19 +1,32 @@
 # kArmor
 
-**kArmor** is a CLI client to help manage [KubeArmor](github.com/kubearmor/KubeArmor)
+**kArmor** is a CLI client to help manage [KubeArmor](github.com/kubearmor/KubeArmor).
 
 KubeArmor is a container-aware runtime security enforcement system that
 restricts the behavior (such as process execution, file access, and networking
 operation) of containers at the system level.
 
 ## Installation
 
+The following sections show how to install the kAmor. It can be installed either from source, or from pre-built binary releases.
+
+### From Script
+
+kAmor has an installer script that will automatically grab the latest version of kAmor and install it locally.
+
 ```
 curl -sfL https://raw.githubusercontent.com/kubearmor/kubearmor-client/main/install.sh | sh
 ```
 
-To build and install, clone the repository and
+The binary will be installed in `<current_dir>/bin` folder.
```

Comment:
Perhaps we should let the user know how to configure the path with `-s -- -b <path>/bin`

```sh
curl -sfL https://raw.githubusercontent.com/kubearmor/kubearmor-client/main/install.sh | sudo sh -s -- -b /usr/local/bin # Needed sudo cause /usr/local is priviledged path

# or 

curl -sfL https://raw.githubusercontent.com/kubearmor/kubearmor-client/main/install.sh | sudo sh -s -- -b $(go env GOPATH)/bin

```